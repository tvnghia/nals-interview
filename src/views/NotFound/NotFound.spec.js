import { shallowMount } from '@vue/test-utils'
import wrapperOps from '@/config-test/wrapper'
import NotFound from '@/views/NotFound'

describe('NotFound.vue', () => {
  const wrapper = shallowMount(NotFound, wrapperOps)

  describe('when NotFound was mounted', () => {
    it('should render correctly', () => {
      expect(wrapper.exists()).toBeTruthy()
      expect(wrapper).toMatchSnapshot()
    })
  })
})
