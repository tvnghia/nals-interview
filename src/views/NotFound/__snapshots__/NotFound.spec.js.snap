// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound.vue when NotFound was mounted should render correctly 1`] = `<p>Page not found</p>`;
